src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -65,4 +66,10 @@ public ResponseEntity addSection(@PathVariable("lineId") Long lindId, @RequestBo
         lineService.addSection(lindId, sectionRequest);
         return ResponseEntity.created(URI.create(format("/%s/sections", lindId))).build();
     }
+
+    @DeleteMapping("/{lineId}/sections")
+    public ResponseEntity deleteSection(@PathVariable("lineId") final Long lineId, @RequestParam("stationId") final Long stationId) {
```

Comment:
새로 만들어진 컨트롤러의 API 에도 제네릭 타입 설정을 해주시면 좋을 것 같습니다 :)

```suggestion
    public ResponseEntity<Void> deleteSection(@PathVariable("lineId") final Long lineId, @RequestParam("stationId") final Long stationId) {
```

Reply:
Void타입이 있는지 처음 알았습니다 😱👍